core/templates/pages/story-editor-page/editor-tab/story-editor.component.html

```
@@ -284,64 +358,143 @@ <h3 class="story-card-header">Story Card</h3>
+                            <div class="chapter-edit-option e2e-test-delete-chapter-button"
+                                 (click)="deleteNode(node.getId());$event.stopPropagation()">
+                              <span>Delete</span>
+                            </div>
                           </div>
-                        </div>
-                      </td>
-                    </tr>
+                        </td>
+                      </tr>
+                    </ng-container>
+                    </ng-container>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </ng-container> ]

(tag-pair)